src/yaml/ingestionTrigger/partData/rasterLayerPartData.yaml

```diff
@@ -0,0 +1,82 @@
+openapi: "3.0.1"
+info:
+  title: Layer polygon part
+  version: 1.0.0
+components:
+  schemas:
+    polygonPartMetadata:
+          type: object
+          description: Layer polygon part data definitions
+          required:
+            - name
+            - imagingTimeBeginUTC
+            - imagingTimeEndUTC
+            - geometry
+            - resolutionDegree
+            - resolutionMeter
+            - sourceResolutionMeter
+            - horizontalAccuracyCE90
+            - sensors
+          properties:
+            id:
+              type: string
+              description: Layer's external identifier
+              pattern: ^[a-zA-Z0-9_-]+$
+            name:
+              type: string
+              description: Layer's external name
+            imagingTimeBeginUTC:
+              type: string
+              description: Oldest imaging date (UTC)
+              format: date-time
+            imagingTimeEndUTC:
+              type: string
+              description: Latest imaging date (UTC)
+              format: date-time
+            resolutionDegree:
+              type: number
+              minimum: 9.e-8
```

Comment:
please update by the following:
Min res Degree to pixel: 0.703125
Max res Degree to pixel: 1.67638e-7
Min res Meter to pixel: 78273
Max res Meter to pixel: 0.0185

Update also in decorators

Reply:
what do you mean by update decorators?